code/modules/wiremod/shell/module.dm

```diff
@@ -81,6 +81,9 @@
 	/// The name of the module to select
 	var/datum/port/input/module_to_select
```

Comment:
subtype this as an option port so that you don't have to do any typecasting.

Reply:
done